src/components/project/project-month-list/ProjectMonthTimesheetCell.tsx

```diff
@@ -128,18 +136,30 @@ export const ProjectMonthTimesheetCell = ({projectMonth}: ProjectMonthTimesheetC
         <NotesModalButton
           value={timesheet.note}
           onChange={val => saveTimesheet({...timesheet, note: val})}
-          title={t('projectMonth.timesheetNote', {name: `${projectMonth.consultant.firstName} ${projectMonth.consultant.name}`})}
+          title={t('projectMonth.timesheetNote', {name: `${fullProjectMonth.consultant.firstName} ${fullProjectMonth.consultant.name}`})}
         />
-        {!projectMonth.invoice && (
+        <div className="timesheet-attachment-actions">
           <UploadFileButton
-            onUpload={f => dispatch(projectMonthUpload(f, 'timesheet', projectMonth._id))}
+            onUpload={f => dispatch(projectMonthUpload(f, 'timesheet', fullProjectMonth._id))}
             icon="fa fa-upload"
             title={t('projectMonth.timesheetUpload')}
+            disabled={!!fullProjectMonth.invoice}
+            style={{
+              borderTopRightRadius: '0',
+              borderBottomRightRadius: '0',
+              borderRight: '0',
+            }}
+          />
+          <AttachmentPreviewButton
+            disabled={!hasTimesheetBeenUploaded}
+            tooltip="projectMonth.viewTimesheet"
+            downloadUrl={getTimesheetDownloadUrl()}
+            style={{
+              borderTopLeftRadius: '0',
+              borderBottomLeftRadius: '0',
+            }}
```

Comment:
As per mijn comment op Slack dat ik toen direct gezegd heb en je daarna nog eens aan herinnert:
Check de InvoiceList pagina --> dit wordt daar gedaan
Gebruik de `btn-group` in plaats van zelf inline styling te gaan doen. Verwijder alle inline styling en gebruik de bootstrap classes die ervoor dienen.